### ISSUE TITLE
less2css.org is expired

### ISSUE THREAD

**Issue by seven-phases-max** [less/less-docs#466]
Just a notification.
(onestly I have no idea what to do with it. Probably it should be moved to lesscss.org/less-preview or so anyway, though it will be sad to loose hundreds of external snippet-links pointing there).

**Comment by matthew-dean** [comment]
Yeah, it should be part of the main site build so that the versions can be more easily kept up-to-date.
